stdlib/REPL/src/docview.jl

```diff
@@ -137,8 +139,6 @@ function _trimdocs(md::Markdown.MD, brief::Bool)
     return Markdown.MD(content, md.meta), trimmed
 end
 
-_trimdocs(md, brief::Bool) = md, false
-
```

Comment:
Don't we still need this? `Markdown.MD` can have non-`Markdown.MD` content.

Reply:
Oh, can it? I didn't actually run this code — just made the "obvious" (bad assumption...) change. I'm not familiar with the Markdown data structures.

Reply:
Yup, you're right. Fixed.